### DISCUSSION TITLE
Reaction-based Roll Assignment

### DISCUSSION THREAD

**Discussion by Akshu-on-github** [EddieHubCommunity/support#1066] [Ideas]
What would you folks think about reaction based roll allotment? (*see reference: `carlbot`, `ticketbot`*)

Possibilities for implementation:
- [ ] Pronouns (with Eddiebot paying attention to individual based choices of, let's say she/her, he/him, they/them, ask me, call me by name, etc.)
- [ ] Roles (would be easier, more intuitive and would take up much less space than the `^iam` system)
- [ ] Opensource reminders (a thumbs up emoji could trigger a destruct of the message - again, takes up much less space)
- [ ] Polls (easy decision making, YouTube polls could be carried over/reflected here)

**Comment by eddiejaoude** [comment]
Interesting suggestions. I will take a look at those bots 👍
